### PR TITLE
Sanitizer

### PR DESCRIPTION
**Pull request by appashchenko**
Fix various problems found by sanitizer.

### CODE REVIEW

File: src/application.c [hunk 1]
```diff
@@ -380,9 +380,8 @@ bool app_init(const char** sources, size_t num)
     }
     if (ctx.window.width == SIZE_FULLSCREEN) {
         ui_toggle_fullscreen();
-    } else if (first_image &&
```
Comment (artemsen) [inline src/application.c:383]: `first_image` always exists, see line 373.

File: src/config.c [hunk 2]
```diff
@@ -312,7 +316,7 @@ bool config_to_color(const char* text, argb_t* color)
     if (strlen(text) > 6) { // value with alpha (RRGGBBAA)
         *color = (num >> 8) | ARGB_SET_A(num);
     } else {
-        *color = num | ARGB_SET_A(0xff);
+        *color = num | ARGB_SET_A(0xffu);
```
Comment (artemsen) [inline src/config.c:319]: It makes the code less readable. Please, remove these `u`s.

File: src/application.c (hunk 1 above)
Comment (appashchenko) [reply src/application.c:383]: > `first_image` always exists, see line 373.

`first_image` can not be NULL after line 374 so it is always true at line 383.

Comment (artemsen) [reply src/application.c:383]: Exactly! So, we don't need to check it twice =)

Comment (appashchenko) [reply src/application.c:383]: Артём, вы не выспались, или я что-то не понимаю?
Коммит наоборот **убирает** эту переменную из условия.

**Comment by artemsen** [comment]
Видимо не выспался =) Да, тут всё норм.

File: src/config.c (hunk 2 above)
Comment (appashchenko) [reply src/config.c]: You are right. it was bad decision.
I have reverted this part and replaced 'u' with cast to argb_t where is needed.
Please review changed commit.

**Pull request merged**